packages/template-retail-react-app/package.json

```diff
@@ -1,5 +1,5 @@
 {
-    "name": "scaffold-pwa",
```

Comment:
The change will cause Circle CI to push a bundle to the MRT with the environment id `retail-react-app`.
But the PWA Kit environment lives in the environment id `scaffold-pwa`.

- **pwa bundle push**
https://github.com/SalesforceCommerceCloud/pwa-kit/blob/1c809ddda24ace45607f9e466f5b68159c71695b/.circleci/config.yml#L275

- **Generated bundle push**
https://github.com/SalesforceCommerceCloud/pwa-kit/blob/1c809ddda24ace45607f9e466f5b68159c71695b/.circleci/config.yml#L356



https://runtime.commercecloud.com/mobify/scaffold-pwa

Reply:
good catch Adam, for some reason i thought we fixed the dependency of package.json name... let me see if we can update the bundle push part to pass the project name.

Reply:
This was most likely the reason why we never changed it :(

Reply:
it looks like `npm run push -- -s scaffold-pwa -m "test"` worked

Reply:
@adamraya @bendvc I just pushed a commit to update CircleCI command to push to the right project. Can you review?

Reply:
👍